x-pack/plugins/lens/public/xy_visualization/expression.tsx

```diff
@@ -206,6 +215,40 @@ export const getXyChartRenderer = (dependencies: {
   },
 });
 
+function mergeThemeWithValueLabelsStyling(
+  theme: RecursivePartial<Theme>,
+  valuesLabelMode: string = 'hide',
+  isHorizontal: boolean
+) {
+  const VALUE_LABELS_MAX_FONTSIZE = 15;
+  const VALUE_LABELS_MIN_FONTSIZE = 10;
+  const VALUE_LABELS_VERTICAL_OFFSET = -10;
+  const VALUE_LABELS_HORIZONTAL_OFFSET = 10;
+
+  if (valuesLabelMode === 'hide') {
+    return theme;
+  }
+  return {
+    ...theme,
+    ...{
+      barSeriesStyle: {
+        ...theme.barSeriesStyle,
+        displayValue: {
+          fontSize: { min: VALUE_LABELS_MIN_FONTSIZE, max: VALUE_LABELS_MAX_FONTSIZE },
+          fill: { textInverted: true, textBorder: true },
```

Comment:
I've ran some tests here with current styling, no border or increased border size.

### Current border

<img width="764" alt="Screenshot 2020-11-02 at 12 33 27" src="https://user-images.githubusercontent.com/924948/97864164-7718f400-1d08-11eb-8bd2-4d1698704d4d.png">
<img width="756" alt="Screenshot 2020-11-02 at 12 38 26" src="https://user-images.githubusercontent.com/924948/97864181-7b451180-1d08-11eb-8b7b-e40ca4950465.png">
<img width="762" alt="Screenshot 2020-11-02 at 12 47 10" src="https://user-images.githubusercontent.com/924948/97865158-0d014e80-1d0a-11eb-9843-7b9605512879.png">


### Thicker border

<img width="764" alt="Screenshot 2020-11-02 at 12 36 21" src="https://user-images.githubusercontent.com/924948/97864201-839d4c80-1d08-11eb-93b0-3b316e892fad.png">
<img width="792" alt="Screenshot 2020-11-02 at 12 37 07" src="https://user-images.githubusercontent.com/924948/97864212-87c96a00-1d08-11eb-9dc4-9fffda47f300.png">
<img width="762" alt="Screenshot 2020-11-02 at 12 50 34" src="https://user-images.githubusercontent.com/924948/97865176-12f72f80-1d0a-11eb-8f1b-205af2da439b.png">


### No border

<img width="762" alt="Screenshot 2020-11-02 at 12 34 37" src="https://user-images.githubusercontent.com/924948/97864237-8dbf4b00-1d08-11eb-8d0d-25a23f35afd9.png">
<img width="763" alt="Screenshot 2020-11-02 at 12 37 42" src="https://user-images.githubusercontent.com/924948/97864256-9283ff00-1d08-11eb-9315-b1363e443ac9.png">
<img width="762" alt="Screenshot 2020-11-02 at 12 47 50" src="https://user-images.githubusercontent.com/924948/97865191-168ab680-1d0a-11eb-938a-cea6285de169.png">


Personally I find the thicker version easier to read compared to both current state and no border at all (exp. with bars with lower contrast with the text). But I see no big readability gain for few bars or vertical charts with many bars.

Reply:
@dej611 Thanks for showing the options, I share your opinion that the thicker border is the easiest of these options. Will you submit a PR to elastic-charts for this? cc @markov00

Reply:
No need for a PR on the `elastic-chart` side.
The `textBorder` already accepts a `number` to increase the size of it. Will update this PR right now.

Reply:
I was mostly suggesting it because I think having good defaults is important, and this looks like the kind of thing we would want to set by default for elastic-charts.

Reply:
I'll open an issue on the repo

Reply:
elastic-chart issue: https://github.com/elastic/elastic-charts/issues/886

Reply:
There are other considerations we should consider before baking this into Elastic Charts. I've commented separately on that issue.